.gitignore

```diff
@@ -28,4 +28,4 @@ coverage/
 cdk.context.json
 .cdk.staging/
 cdk.out/
-
+*.tabl.json
```

Comment:
do we want this in pkglint?

Reply:
It's just going to be created here, not in any subpackage. So not really?